static/app/utils/useNavigate.tsx

```diff
@@ -37,10 +36,14 @@ export function useNavigate(): ReactRouter3Navigate {
 
     // biome-ignore lint/correctness/useHookAtTopLevel: react-router-6 migration
     const navigate = useCallback<ReactRouter3Navigate>(
-      (to: LocationDescriptor | number, options: NavigateOptions = {}) =>
-        typeof to === 'number'
-          ? router6Navigate(to)
-          : router6Navigate(locationDescriptorToTo(to), options),
+      (to: LocationDescriptor | number, options: NavigateOptions = {}) => {
+        if (typeof to === 'number') {
+          router6Navigate(to);
+          return;
+        }
+
+        router6Navigate(normalizeUrl(to), options);
```

Comment:
Does `normalizeUrl` do the `locationDescriptorToTo` stuff?

Reply:
Oops no it does not! Will add back